activestorage/lib/active_storage/service/gcs_service.rb

```diff
@@ -35,7 +35,11 @@ def download(key)
 
     def delete(key)
       instrument :delete, key do
-        file_for(key).try(:delete)
+        begin
+          file_for(key).delete
```

Comment:
We still need the `try`, no? Won’t `file_for` potentially return `nil`?

Reply:
It seems not. We have a test that covers deleting a nonexistent key.

Reply:
I thought it wasn't necessary, but thinking again about it, I think it doesn't hurt and it's safer to leave it. I'll amend the commit.

Reply:
I forgot that the cloud service tests don’t run for PRs. We do need to keep it. :+1:

Reply:
Oh, interesting! I run `activestorage` tests locally before creating this PR and they passed. I wonder if the test is not really testing this...

Reply:
Ah, never mind, I had ignored this, hehehe
```Skipping S3 Direct Upload tests because no S3 configuration was supplied
Skipping GCS Direct Upload tests because no GCS configuration was supplied
Skipping Azure Storage Direct Upload tests because no Azure Storage configuration was supplied
Skipping Azure Storage Service tests because no Azure configuration was supplied
Skipping GCS Service tests because no GCS configuration was supplied
Skipping S3 Service tests because no S3 configuration was supplied
```